src/test/java/jek/collections/IntArrayListTest.java

```diff
@@ -107,10 +135,29 @@ void testRemove() {
 
     @Test
     void indexOf() {
+        IntArrayList ial = new IntArrayList();
+        ial.add(1);
+        ial.add(5);
+        ial.add(54);
+        ial.add(11);
+        ial.add(75);
+        ial.add(59);
+
+        assertEquals(3, ial.indexOf(11));
     }
 
     @Test
     void lastIndexOf() {
+        IntArrayList ial = new IntArrayList();
+        ial.add(1);
+        ial.add(5);
+        ial.add(54);
+        ial.add(11);
+        ial.add(5);
+        ial.add(59);
+        ial.add(54);
+
+        assertEquals(4, ial.lastIndexOf(5));
     }
```

Comment:
When you are storing or retrieving the object always think how do you want your application to behave when you are passing the null value. Test it also please. 
What should there be if there's no such element in your list.